Add end user restore documentation (#3045)
codenvy/che-docs: modules/end-user-guide/pages/proc-restoring-a-workspace-from-backup.adoc

> :mod-docs-content-type: PROCEDURE

Reviewer: **Attribute name typo**: Missing leading underscore. Vale `ContentType` will flag this.

```suggestion
:_mod-docs-content-type: PROCEDURE
```

> [id="restoring-a-workspace-from-backup{context}"]

Reviewer: **Missing `_` before `{context}`**: The ID anchor needs an underscore separator before `{context}`.

```suggestion
[id="restoring-a-workspace-from-backup_{context}"]
```

= Restoring a workspace from a backup

> You can start a new workspace from an existing backup by using the {prod-short} dashboard. When the restoration is complete, the dashboard opens the new workspace in a browser tab.

Reviewer: ```suggestion
Recover uncommitted changes or recreate a workspace environment by restoring from a backup snapshot through the {prod-short} dashboard. When the restoration is complete, the dashboard opens the new workspace in a browser tab.
```

.Prerequisites

> 

Reviewer: **Prerequisites must use unordered list** (`*`), not ordered list (`. `). Per the Red Hat modular docs template, prerequisites are always bulleted.

```suggestion
* Backups are enabled for your cluster.
```

> . Backups are enabled for your cluster.

Reviewer: Same — unordered list for prerequisites:

```suggestion
* You know which default registry or external registry backup image to use.
```

. You have identified a default registry or external registry backup image to use.

.Procedure

. On the Restore page, select the backup source:
** Default registry: Select a backup from the list.
** External registry: Provide a backup image URL.
+
.Restore mode and backup source selection
image::workspace-backup/restore_section_1.png[Selecting the backup source during restoration,link="{imagesdir}/workspace-backup/restore_section_1.png"]
. Enter a name for the restored workspace.
. Optional: Configure the memory limits, CPU limits, and the editor.
+
.Resource limits and editor selection
image::workspace-backup/restore_section_2.png[Configuring resources and editors for a restored workspace,link="{imagesdir}/workspace-backup/restore_section_2.png"]

> . Click Restore Workspace and wait for the workspace to start.

Reviewer: ```suggestion
. Click *Restore Workspace* and wait for the workspace to start.
```
